jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -153,7 +158,7 @@ public static Configuration fromEnv() {
     Metrics metrics = new Metrics(statsFactory);
     Reporter reporter = reporterConfig.getReporter(metrics);
     Sampler sampler = samplerConfig.createSampler(serviceName, metrics);
-    return new Tracer.Builder(serviceName, reporter, sampler).withMetrics(metrics);
+    return initTracerTags(new Tracer.Builder(serviceName, reporter, sampler).withMetrics(metrics));
```

Comment:
I think it would be cleaner to do something like

```java
return new Tracer.Builder(serviceName, reporter, sampler).withMetrics(metrics).withTags(tagsFromEnv());
```